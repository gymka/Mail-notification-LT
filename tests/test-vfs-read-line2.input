first line
second line
third line (terminated)
