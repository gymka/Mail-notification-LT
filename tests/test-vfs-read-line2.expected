line: first line
line: second line
line: third line (terminated)
